packages/design-system/src/components/Tag/variations/TagDefault.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef, HTMLAttributes, Ref } from 'react';
+
+import TagPrimitive from '../primitive';
+
+import style from './TagDefault.module.scss';
+
+type TagProps = Omit<HTMLAttributes<HTMLSpanElement>, 'className'>;
```

Comment:
```suggestion
type TagProps = Omit<BaseTagProps, 'className'>;
```
There's a strong chance the primitive types might change and we would want to inherit them.